jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -119,7 +128,7 @@ public void executeTimerJob(Timer timer) {
         try {
             transaction(this::executeTimerJobInstance, timerJobInstance);
         } catch (Exception e) {
-            recoverTimerJobInstance(timerJob, e);
+            recoverTimerJobInstance(timer, timerJob, e);
```

Comment:
timer is not used in recoverTimeJobInstance

Reply:
removed